careers/blocks/slider/slider.css

```diff
@@ -278,6 +282,11 @@
   .slider.spacious .card-wrapper {
     gap: 50px;
   }
+
+  .slider.max-visible-2.block .card-wrapper {
+    width: calc(100%/2 - 4px);
```

Comment:
Isn't 100%/2 the same as 50%?

Reply:
Yes. Keeping that calculation for clarity. For 3 cards, it's 100%/3.